examples/shaders/resources/shaders/glsl120/raymarching.fs

```diff
@@ -0,0 +1,431 @@
+//Ported by CDM to work on older hardware.
+#version 120
+
```

Comment:
Why not define `precision mediump float`?

Reply:
if it's for 100 ES yes, but regular 100/120 no.